Week3/hw/app13.js

```diff
@@ -0,0 +1,9 @@
+//  13. Create an object that contains the teachers that you have had so far for the different modules.
+
+const instructors = {
+  a: { name: 'Philipp' },
+  b: { name: 'Rob' },
+  c: { name: 'Unmesh' },
+  d: { name: 'Bonan' },
+  e: { name: 'Yash' },
```

Comment:
awesome! I would only suggest to change the `keys` of the objects to more descriptive ones. For example:

```
const instructors = {
  philipp: { name: 'Philipp' },
  rob: { name: 'Rob' },
...
}
```

Reply:
Edited..